### PR TITLE
Updating Setting Up Python Environment Document

### PR DESCRIPTION
**Pull request by JessDF**
**1. Brief Summary of what this PR accomplishes (140 characters or less. If you find trouble describing what you are doing in this length, consider breaking the PR into multiple ones.)**
Included error within documentation to demonstrate when one might need to update the environment.yml page.

**2. Link to Trello Ticket**
No ticket, however, was discussed within the #python slack channel: https://sfbrigade.slack.com/archives/C24LU6QMP/p1520294340000405?thread_ts=1520294214.000137&cid=C24LU6QMP

**3. More detailed description and other questions to address in code review**
Is this example laid out well enough? or should I change the wording?

**4. Remember to tag reviewers!**

### CODE REVIEW

File: onboarding/02_development_environment.md [hunk 1]
```diff
@@ -92,6 +92,13 @@ For further information, here is a useful guide to conda environments: https://c
 
 Note, the [environment.yml](../environment.yml) file must be kept up to date and is how we will ensure that every group member is on the same environment so any work we do on any machine is reproducible on any other machine.
 
+If you have acitvate your python environment and recieve the following error, you may need to update your python environment:<br>
```
Comment (erikeldridge) [inline onboarding/02_development_environment.md]: typo: acitvate --> activated

What are the steps to "update your python environment"? If it's via `conda env update`, you could append something like "... update your python environment using `conda env update` as described above".

Comment (JessDF) [reply onboarding/02_development_environment.md]: Can I make these changes and have them on the same pull request or do I need to create a new one?

Comment (erikeldridge) [reply onboarding/02_development_environment.md]: Let's try to do it in this same PR. I'm happy to re-approve, but I hope our repo is config'd to tolerate small changes.

Comment (erikeldridge) [reply onboarding/02_development_environment.md]: Looks like I might have to re-review, per "If required reviews are enabled and a collaborator ... submits a review requesting changes, the pull request cannot be merged until the same collaborator submits another review approving the changes in the pull request." – https://help.github.com/articles/reviewing-proposed-changes-in-a-pull-request/

**Comment by JessDF** [comment]
@erikeldridge Made the new commit, and it looks like it was added to this pull request. If it looks good to you, I will merge it.

**Pull request merged**